src/components/ArtistPieChart/ArtistPieChart.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useEffect, useState } from 'react';
+import getRecommendedArtists from '../../services/getRecommendedArtists';
+import './ArtistPieChart.scss';
+
+const getBrands = (brands : []) : string => brands.map((val : {name : string}) => val.name).join(', ');
+const getAge = (age : {ageGroup : string, matchPercentage : number}) : string => `${age.ageGroup}, ${age.matchPercentage}`;
+const getGender = (gender : {male : number, female : number}) : string => {
+  if (gender.male > gender.female) return `Male, ${gender.male}`;
+  if (gender.female > gender.male) return `Female, ${gender.female}`;
+  return 'Both, 50';
+};
+const getLocations = (locations : []) : string => locations.map((val : {name : string}) => val.name).join(', ');
+const ArtistPieChart = () : JSX.Element => {
+  const [brand1, setBrand1] = useState('');
+  const [brand2, setBrand2] = useState('');
+  const [brand3, setBrand3] = useState('');
+  const [brand4, setBrand4] = useState('');
+  const [name1, setName1] = useState('');
+  const [name2, setName2] = useState('');
+  const [name3, setName3] = useState('');
+  const [name4, setName4] = useState('');
+  const [match1, setMatch1] = useState('');
+  const [match2, setMatch2] = useState('');
+  const [match3, setMatch3] = useState('');
+  const [match4, setMatch4] = useState('');
+  const [img1, setImg1] = useState('');
+  const [img2, setImg2] = useState('');
+  const [img3, setImg3] = useState('');
+  const [img4, setImg4] = useState('');
+  const [age1, setAge1] = useState('');
+  const [age2, setAge2] = useState('');
+  const [age3, setAge3] = useState('');
+  const [age4, setAge4] = useState('');
+  const [gender1, setGender1] = useState('');
+  const [gender2, setGender2] = useState('');
+  const [gender3, setGender3] = useState('');
+  const [gender4, setGender4] = useState('');
+  const [location1, setLocation1] = useState('');
+  const [location2, setLocation2] = useState('');
+  const [location3, setLocation3] = useState('');
+  const [location4, setLocation4] = useState('');
+
+  const getData = async () => {
+    const data = await getRecommendedArtists('871717');
+    const { artists } = data;
+    const [artist1, artist2, artist3, artist4] = artists;
+    // eslint-disable-next-line
+    console.log(artist1);
```

Comment:
remove the console.
The below code doesnt look promising. We can avoid such repeatations